test/testcafe/framework/page-objects/SignInDevicePageObject.js

```diff
@@ -0,0 +1,36 @@
+import { Selector } from 'testcafe';
+import BasePageObject from './BasePageObject';
+
+export default class SignInDeviceViewPageObject extends BasePageObject {
+  constructor(t) {
+    super(t);
+    this.t = t;
+    this.beacon = new Selector('.beacon-container');
+    this.body = new Selector('.launch-authenticator');
+    this.footer = new Selector('.auth-footer');
+  }
+
+  getBeaconClass() {
```

Comment:
Do you need to verify Beacon in test as well?

Reply:
No, not really. I will remove the method.

Reply:
Looks like the view does have beacon? I thought you would need to add assertion